Form/FieldTypeHandler/EnhancedSelection.php

```diff
@@ -67,7 +71,7 @@ protected function buildFieldForm(
         $fieldSettings = $fieldDefinition->getFieldSettings();
         $optionsValues = $fieldSettings['options'];
 
-        $options['expanded'] = false;
+        $options['expanded'] = true;
```

Comment:
I know that, but it's not related to this PR, and moreover, legacy type displays `<select>` element and does not have the option to display checkboxes and radio buttons, so we should keep it consistent.